web/EXAMPLES.md

```diff
@@ -0,0 +1,166 @@
+# Using DuckDuckGo's Web Tracker Blocklist
+
+The following description explains, at the high-level, how web blocklists are used for tracker blocking in DuckDuckGo apps and extensions.
+
+For each third party asset request on the site, take the hostname of a request, including subdomains:
+
+1. Check for a [cloaked CNAME](https://help.duckduckgo.com/duckduckgo-help-pages/privacy/web-tracking-protections/#cname-cloaking-protection) by looking for a corresponding key in the [cnames](https://github.com/duckduckgo/tracker-blocklists/web/README.md#cnames) object. Continue removing subdomains until a match is found or you no longer have a valid hostname. 
+
+    - If a match is found replace the request hostname with the matched `cnames` value. Continue to step 2 with the updated request.
+    - If no match is found continue to step 2 with the unaltered request.
+
+2. With the third party request from 1, check for a corresponding key in the [trackers](https://github.com/duckduckgo/tracker-blocklists/web/README.md#trackers) object. Continue removing subdomains until a match is found or you no longer have a valid hostname. 
+
+    - If no match is found, don't block the request
+    - If a match is found, continue to step 3
+
+3. If a match is found:
+
+- 3a. If the entry has rules
+    - Rules are ordered from most specific to least specific, and only one rule can match per request.
+   
+    - Loop through the rule list and apply regular expression, ignoring case and start/end position, with the full request URL using each rule.
+        - If a rule match is found, then for the first matching rule:
```

Comment:
why all text below is in a codeblock?
![codeblock](https://user-images.githubusercontent.com/985504/186713369-9aed0bfb-4cb4-48a7-abdc-1ffb22eddef6.jpg)

Reply:
I think we have hit some nested list limit in markdown. I've given up trying to get this to render as a normal text list.

Reply:
Switching to a,b,c fixed it 🤷‍♂️